pkg/apis/scorecard/v1alpha3/doc.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +k8s:deepcopy-gen=package,register
+// +groupName=osdk.openshift.io
```

Comment:
```suggestion
// +groupName=operators.operatorframework.io
```

Reply:
I was under impression that we are keeping groupName similar to existing `v1alpha2`.

Reply:
operators.operatorframework.io is what the enhancement PR specifies currently,   the website is sdk.operatorframework.io, so I'm not sure what the standard group name should be.  Joe what do you think?  @joelanford

Reply:
I think `scorecard` should be somewhere in the group name.

My thought would be either of these:
- scorecard.operatorframework.io
- scorecard.sdk.operatorframework.io

I think I lean toward the first just because of the length of the string and the fact that I don't see it overlapping with anything else in the operator framework.

As an aside, I think we need to do a full scrub of domains used across operator-sdk and see if we can come up with some semblance of consistency and a pattern to follow when we need a new domain/group name.